unirec/unirec.c

```diff
@@ -286,18 +286,19 @@ const char *ur_get_type_and_name_from_string(const char *source, char **name, ch
 {
    int length_type_2 = 0, length_name_2 = 0;
    const char *source_cpy;
-   source_cpy = source;
-   /* skip white spaces */
+   /* skip spaces */
```

Comment:
Previously there were both variants used and I just wanted to unify it, but you're right that "white space" is better. I'll fix it.